FrontMen/fm-hours
# feat: adding composable for employees

# Changes

I have create a 'useEmployees' composable which we can re-use in every vue component if we want to retrieve
- a list the employes
- a list of all employess as items for a table
- a function with which we can retrieve an employee by **id**
- a computed which returns the employee based on the employee id route param

## Added



## How to test

- You should be able to see the employees list with all the employees when navigating to the Employees page. 
- When navigating to the Timesheets page, you should be able to see the employees list in the left side bar as well as clicking to a specific box(check screenshot below) next to the employee name, should show the calendar of the employee's hours for current week in the right side of the page.

## Screenshots
<img width="1728" alt="Screenshot 2023-01-02 at 11 35 59" src="https://user-images.githubusercontent.com/64504654/210221321-5bd53833-0fa1-4579-9993-63430cebbcf2.png">
